src/content/getting-started/provision-your-first-workload-cluster/_index.md

```diff
@@ -127,95 +129,45 @@ kubectl gs template cluster \
-
-- `App/<cluster>`
-- `ConfigMap/<cluster>-userconfig`
-
-If you would like to protect your clusters from accidental deletion, take a look at our [deletion prevention mechanism]({{< relref "/vintage/advanced/app-platform/deletion-prevention" >}}).
-
-### Private workload clusters
-
-By default, the created Kubernetes cluster API endpoint is public. See [Private clusters]({{< relref "/vintage/advanced/cluster-management/private-clusters" >}}) if you want to limit networking to/from the cluster.
-
-## Step 4: Watch the status of workload clusters
-
-In the Web UI, click on the cluster's name to see its details. The workload cluster's name is shown on the top-left (in the screenshot: `o8r3r`).
-
-![Adding a nodepool](nodepool.png)
-
-For the vintage product family, if you didn't yet attach a node pool to the WC as shown above, you can now add one by clicking the "Add node pool" button.
-
-Using the command line, you can also watch the creation and status of the workload cluster:
-
-{{< tabs >}}
-{{< tab id="status-vintage" for-impl="vintage_any">}}
-
-```sh
-kubectl gs get clusters -A
-
-kubectl gs get nodepools -A
 ```
 
-{{< /tab >}}
-{{< tab id="status-capi" for-impl="capi_any">}}
+## Step 2: Watch the status of the workload cluster
 
-Either use kubectl-gs:
+You can watch the creation and status of the workload cluster running:
 
 ```sh
 kubectl gs get clusters -A
 ```
 
-Or use other kubectl tooling to display the relations and status of the CAPI manifests:
+Additionally, you may want to display the relations and status of the Cluster API manifests using upstream tooling:
 
 ```sh
 kubectl describe clusters.cluster.x-k8s.io -n org-testing name-of-workload-cluster
 
-# Using CAPI's clusterctl tool (https://cluster-api.sigs.k8s.io/clusterctl/overview.html)
+# Using Cluster API's clusterctl tool (https://cluster-api.sigs.k8s.io/clusterctl/overview.html)
 clusterctl describe cluster -n org-testing name-of-workload-cluster --show-conditions all
 
 # Using kubectl-tree plugin (https://github.com/ahmetb/kubectl-tree)
 kubectl krew install tree
 kubectl tree clusters.cluster.x-k8s.io -n org-testing name-of-workload-cluster
 ```
 
-Note how our example commands use the fully qualified CRD name `clusters.cluster.x-k8s.io`. The shorthands `cluster` or `clusters` also work as long as there are no [custom resource definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) with a conflicting name installed.
-
-{{< /tab >}}
-{{< /tabs >}}
-
-## Step 5: Log in to the workload cluster
+Note how our example commands use the fully qualified Custom Resource Definition (CRD) name `clusters.cluster.x-k8s.io`. The shorthands `cluster` or `clusters` also work as long as there are no [custom resource definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) with a conflicting name installed.
```

Comment:
If we know for sure that there will be CRDs with conflicting names, I'd explain it here.

Reply:
yeah, you are right; I don't think it is the case anymore. Looking at a new CAPI management cluster, I see we still have two CRDs with a `cluster` name, but the CAPI one has precedence.
```
clusters.cluster.x-k8s.io            
clusters.rds.aws.upbound.io
```
I will convert this in a warning